src/models/shopping-cart.model.ts

```diff
@@ -25,6 +25,10 @@ export class ShoppingCartItem {
    */
   @property()
   price?: number;
+
+  constructor(item: Partial<ShoppingCartItem>) {
+    Object.assign(this, item);
```

Comment:
Is `super(item);` better?

Reply:
This is not a derived class, there is no super.